### ISSUE TITLE
Show error to user when looking up a livenet address on the testnet site

### ISSUE THREAD

**Issue by gabegattis** [bitpay/insight#686]
I had the testnet insight open and forgot it was the testnet version. I tried to look up a livenet address and it said "There are no transactions involving this address."

Instead, we should detect that the address was for the wrong network and point the user to the livenet insight.

**Comment by braydonf** [comment]
I agree, also would be great to have the header color be different, and display the network in text in the header.

**Comment by Levino** [comment]
Yes, related. Please show an error message, not a "point to live deployment". Maybe insight has been deployed by someone on testnet only. Just show "wrong network - I am testnet!"
